File: SDKs/beezup.com/2.0/typescript/src/sdk/models/shared/lastmanualimportinputconfiguration.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { InputConfiguration } from "./inputconfiguration";



// LastManualImportInputConfiguration
/** 
 * Last manual import input configuration
**/
export class LastManualImportInputConfiguration extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=input" })
  input: InputConfiguration;
}

File: SDKs/beezup.com/2.0/python/src/sdk/models/shared/lastmanualimportinputconfiguration.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import inputconfiguration as shared_inputconfiguration


@dataclass_json
@dataclasses.dataclass
class LastManualImportInputConfiguration:
    r"""LastManualImportInputConfiguration
    Last manual import input configuration
    """
    
    input: shared_inputconfiguration.InputConfiguration = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('input') }})
    
